api/Map.js

```diff
@@ -0,0 +1,209 @@
+// see https://github.com/camptocamp/cgxp/blob/master/core/src/script/CGXP/api/Map.js
+
+import OLMap from 'ol/Map.js';
+import Feature from 'ol/Feature.js';
+import Point from 'ol/geom/Point.js';
+import {Icon, Style} from 'ol/style.js';
+import View from 'ol/View.js';
+import VectorSource from 'ol/source/Vector.js';
+import VectorLayer from 'ol/layer/Vector.js';
+import TileLayer from 'ol/layer/Tile.js';
+
+import {get as getProjection} from 'ol/proj.js';
+
+import * as constants from './constants.js';
+
+// FIXME: temporary
+import SwisstopoSource from '@geoblocks/sources/Swisstopo.js';
+import EPSG_21781 from '@geoblocks/sources/EPSG21781.js';
```

Comment:
Can you use `EPSG21781` as in the rest of the project (to search and replace in the future for instance)

Reply:
done